pages/search/search.pixy

```diff
@@ -64,4 +64,17 @@ component SearchResults(term string, users []*arn.User, animes []*arn.Anime, pos
 					each user in users
 						.mountable(data-mountable-type="user")
 							Avatar(user)
-						//- a.ajax(href=user.Link())= user.Nick
\ No newline at end of file
+						//- a.ajax(href=user.Link())= user.Nick
+		.widget
+			h3.widget-title
+				Icon("user")
+				span Characters
+
+			.user-avatars.user-search
```

Comment:
This is not a list of user avatars so it shouldn't have the class `user-avatars`.
`user-search` should be `character-search`. Scarlet file probably just needs a `horizontal-wrap`.